### PR TITLE
(WSR 8) - <MessageBoxFunctionalLayout/> - remove green theme

### PR DESCRIPTION
**Pull request by EgoziE**
Props:
- removed value `green` in prop `theme` - will fall back to `blue`.

### CODE REVIEW

File: src/MessageBox/FunctionalLayout/HeaderLayout.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ HeaderLayout.propTypes = {
   title: PropTypes.node,
   onCancel: PropTypes.func,
   closeButton: PropTypes.bool,
-  theme: PropTypes.oneOf(['red', 'green', 'blue', 'lightGreen', 'purple']),
+  theme: PropTypes.oneOf(['red', 'blue', 'purple']),
```
Comment (mykas) [inline src/MessageBox/FunctionalLayout/HeaderLayout.js]: I see that HeaderLayout is actually exposed in our index.js but we don't probably advertise this, right? Maybe worth deleting from main index.js  🤔

Comment (mykas) [reply src/MessageBox/FunctionalLayout/HeaderLayout.js]: @shlomitc I don't have full story on our HeaderLayout component on reason why it is exposed. This PR is probably not the place to do it, but maybe we should consider exports like `HeaderLayout`, `HeaderLayout1`, `FooterLayout`, `FooterLayout1`  to be removed: 
Screenshot 2020-03-22 at 15 57 45

Comment (shlomitc) [reply src/MessageBox/FunctionalLayout/HeaderLayout.js]: We don't want to expose them, so from my side, we can just remove it

Comment (EgoziE) [reply src/MessageBox/FunctionalLayout/HeaderLayout.js]: can be done in master I guess

Comment (mykas) [reply src/MessageBox/FunctionalLayout/HeaderLayout.js]: I am not sure about the master. I know at least two teams who uses it.

**Pull request merged**